app/mailers/user_mailer.rb

```diff
@@ -10,4 +10,13 @@ def user_membership_destroyed_notification(user, organisation)
                  subject: t(".subject", organisation_name: organisation.name),
                  body: t(".body", user_name: user.full_name, organisation_name: organisation.name, service_name:)
   end
+
+  def user_claim_submitted_notification(user, claim, link_to_claim)
```

Comment:
```suggestion
  def claim_submitted_notification(user, claim, link_to_claim)
```
According to the article @Nitemaeric send (https://katanacode.com/blog/ruby-on-rails-action-mailer-how-to-name-mailer-methods/), this should be the resource which is changed (`claim`). So you don't need the `user`.
In the existing examples it's because the `UserMembership` is being changed.